src/main/java/com/nutomic/syncthingandroid/service/SyncthingRunnable.java

```diff
@@ -126,31 +127,15 @@ public void run() {
         try {
             if (wakeLock != null)
                 wakeLock.acquire();
+
             ProcessBuilder pb = (mUseRoot)
-                    ? new ProcessBuilder("su", "-c", TextUtils.join(" ", mCommand))
+                    ? new ProcessBuilder("su")
                     : new ProcessBuilder(mCommand);
```

Comment:
You should probably move this into `setupAndLaunch()`, as it doesn't make much sense on its own.

Reply:
Yeah. Fixed.